src/lib/versions.ts

```diff
@@ -62,18 +62,32 @@ function getForcVersion(docsDir: string) {
   };
 }
 
-// function getSwayLibsVersion(docsDir: string) {
-//   const forcfiledir = join(docsDir, 'sway-libs/forc-pkg/Cargo.toml');
-//   const forcfile = fs.readFileSync(forcfiledir, 'utf-8');
-//   const version = forcfile?.match(/version = "(.*)"/)?.[1];
-
-//   return {
-//     name: 'forc',
-//     category: 'Forc',
-//     version,
-//     url: `https://github.com/FuelLabs/sway/tree/v${version}`,
-//   };
-// }
+function getSwayLibsVersion(docsDir: string) {
+  // TODO: uncomment this once sway-libs 0.22.0 is released
```

Comment:
what's important about 0.22.0 release?

Reply:
the PR to release that versions also adds a Cargo.toml file that has the version of the repo. Right now the version isn't in the repo.